# 1. Simulator parameters
use_sim_time: true
clockscale: 1.0                         # only 1.0 is supported yet

# 2. Vehicle initial geodetic position
lat_ref : 55.7544426
lon_ref : 48.742684
alt_ref : -6.5

# 3. Vehicle initial local position
# X, Y, Z, roll, pitch, yaw, weight
init_pose: [0.00, 0.00, 0.0,
            0.00, 0.00, +0.00, +1.00]
